app/person-escort-record/controllers.js

```diff
@@ -0,0 +1,30 @@
+const wizard = require('hmpo-form-wizard')
+
+const FormWizardController = require('../../common/controllers/form-wizard')
+
+function sectionHandler(req, res, next) {
+  const { questions } = req.framework
+  const { key: sectionKey, steps: sectionSteps } = req.frameworkSection
```

Comment:
Rather than passing the framework in via `req`, could you not just pass the frameworkSection wanted, and then require in the framework here?

I think it would also make for simpler mocking.

Reply:
So the purpose of that was for some changes coming in later PRs. Basically we will need to use it in other controllers and setting in a middleware felt like a cleaner way to re-use this object. 

It also means we only need to call the method to load the framework from files once which felt more efficient.